controllers/fenceagentsremediation_controller_test.go

```diff
@@ -333,19 +351,37 @@ func testPodDeletion(podName string, resourceDeletionWasTriggered bool) {
 			err := k8sClient.Get(context.Background(), podKey, pod)
 			return apierrors.IsNotFound(err)
 
-		}, 5*time.Second, 250*time.Millisecond).Should(BeTrue())
+		}, timeoutDeletion, pollInterval).Should(BeTrue())
 		log.Info("Pod is no longer exist", "pod", podName)
 	} else {
 		ConsistentlyWithOffset(1, func() bool {
 			pod := &corev1.Pod{}
 			err := k8sClient.Get(context.Background(), podKey, pod)
 			return apierrors.IsNotFound(err)
 
-		}, 5*time.Second, 250*time.Millisecond).Should(BeFalse())
+		}, timeoutDeletion, pollInterval).Should(BeFalse())
 		log.Info("Pod exist", "pod", podName)
 	}
 }
 
+// verifyStatusCondition checks whether the status condition is set with the expected value
+func verifyStatusCondition(testFAR *v1alpha1.FenceAgentsRemediation, conditionType, expectedResult string, conditionStatus metav1.ConditionStatus) {
```

Comment:
I won't block on this, but using this "expectedResult" string is strange at least. Type and status params should be enough for the tests, not? Either they match or not.

Reply:
Yes, but I added the `expectedResult` so the Eventually can "catch" an expected error, e.g., in [unit-test](https://github.com/medik8s/fence-agents-remediation/pull/69/files#diff-fac0cf56dda42683c1e24b46c87cfa4cca46c444145ee69de9e264db50f10289R201-R204) when the conditions haven't been set.
I have changed the name to `verifyExpectedStatusConditionError` as it better captures the essence of this function.

Reply:
you could make `conditionStatus` a pointer and use `nil`, if you want to test that a condition isn't set 🙂

Reply:
But then the test would fail, since eventually will expect `utils.ConditionSetAndMatchSuccess` while verifyStatusCondition/verifyExpectedStatusConditionError would return `utils.ConditionSetButNoMatchError`.

Reply:
this works: https://github.com/slintes/fence-agents-remediation/commit/55781eac1d0652cfa99f739aeafef21b62d32d18?diff=unified